package.json

```diff
@@ -7,9 +7,10 @@
     "dist/backdraft.js"
   ],
   "scripts": {
+    "eslint": "eslint-git-changes --jsx",
     "setup": "npm install -g grunt-cli",
-    "specs": "karma start --single-run",
-    "dev": "karma start",
+    "specs": "yarn run eslint; karma start --single-run",
```

Comment:
should we instead add this to the karma config so it is always ran on `karma start`? Would DRY this out and seems to be our intended workflow

Reply:
I think the `specs` script will run on CI. Since we're using Hound, should we be running eslint on CI?

Reply:
I tried hooking eslint into our karma workflow but no luck. 
When there is a eslint error, it will continue on and run the tests (this is why we used a ; instead of a &&)

Ideally, don't think we want eslint on CI but given our current setup it will run on there. I suppose we can solve that by having a separate script for our CI...does that sound good?

Reply:
👍

Reply:
script for CI added: https://github.com/Invoca/backdraft/pull/98/commits/59149ada0bad86df1752b361038d620133e672ad